Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -298,54 +155,34 @@ bool ABenchmarkGymGameMode::ShouldUseCustomSpawning()
 
 void ABenchmarkGymGameMode::ParsePassedValues()
 {
-	USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(GetNetDriver());
+	Super::ParsePassedValues();
+
+	PlayerDensity = ExpectedPlayers;
+
 	if (FParse::Param(FCommandLine::Get(), TEXT("OverrideSpawning")))
 	{
-		UE_LOG(LogBenchmarkGym, Log, TEXT("Found OverrideSpawning in command line args, worker flags for custom spawning will be ignored."));
-		FParse::Value(FCommandLine::Get(), TEXT("TotalPlayers="), ExpectedPlayers);
-		// Set default value of PlayerDensity equal to TotalPlayers. Will be overwritten if PlayerDensity option is specified.
-		PlayerDensity = ExpectedPlayers;
 		FParse::Value(FCommandLine::Get(), TEXT("PlayerDensity="), PlayerDensity);
-		FParse::Value(FCommandLine::Get(), TEXT("TotalNPCs="), TotalNPCs);
-
-		FParse::Value(FCommandLine::Get(), TEXT("MaxRoundTrip="), MaxClientRoundTripSeconds);
-		FParse::Value(FCommandLine::Get(), TEXT("MaxLateness="), MaxClientViewLatenessSeconds);
 	}
 	else
 	{
-		UE_LOG(LogBenchmarkGym, Log, TEXT("Using worker flags to load custom spawning parameters."));
-		FString TotalPlayersString, PlayerDensityString, TotalNPCsString, MaxRoundTrip, MaxViewLateness;
+		USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(GetNetDriver());
 		check(NetDriver);
-		if (NetDriver != nullptr && NetDriver->SpatialWorkerFlags != nullptr && NetDriver->SpatialWorkerFlags->GetWorkerFlag(TEXT("total_players"), TotalPlayersString))
-		{
-			ExpectedPlayers = FCString::Atoi(*TotalPlayersString);
-		}
-		// Set default value of PlayerDensity equal to TotalPlayers. Will be overwritten if PlayerDensity option is specified.
-		PlayerDensity = ExpectedPlayers;
-		if (NetDriver != nullptr && NetDriver->SpatialWorkerFlags != nullptr)
+
+		UE_LOG(LogBenchmarkGymGameModeBase, Log, TEXT("Using worker flags to load custom spawning parameters."));
+		FString PlayerDensityString;
+
+		const USpatialWorkerFlags* SpatialWorkerFlags = NetDriver != nullptr ? NetDriver->SpatialWorkerFlags : nullptr;
+		if (SpatialWorkerFlags != nullptr)
```

Comment:
Could be worth adding a check here, we require these to be set or the test will be invalid
```suggestion
		checkf(SpatialWorkerFlags, "Worker flags are requried to be set");
		if (SpatialWorkerFlags != nullptr)
```